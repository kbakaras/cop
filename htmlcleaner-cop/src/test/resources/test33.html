<script type="text/javascript">/*<![CDATA[*/
function escapeForXML(origtext) {
   return origtext.replace(/\\&/g,'&'+'amp;').replace(/</g,'&'+'lt;')
       .replace(/>/g,'&'+'gt;').replace(/\'/g,'&'+'apos;')
       .replace(/\"/g,'&'+'quot;');}
/*]]>*/</script>