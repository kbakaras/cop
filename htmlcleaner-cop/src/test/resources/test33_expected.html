<?xml version="1.0" encoding="UTF-8"?>
<html><head /><body><script type="text/javascript">/*<![CDATA[*/
function escapeForXML(origtext) {
   return origtext.replace(/\\&/g,'&'+'amp;').replace(/</g,'&'+'lt;')
       .replace(/>/g,'&'+'gt;').replace(/\'/g,'&'+'apos;')
       .replace(/\"/g,'&'+'quot;');}
/*]]>*/</script></body></html>
